### DISCUSSION TITLE
Struggling with extractVersion

### DISCUSSION THREAD

**Discussion by gregoryduckworth** [renovatebot/renovate#8068] [Request Help] [answered]
HI,

I've been banging my head against a wall trying to parse the versioning coming from `github-tags` for `nginx`. They contain `release-` at the start however I am only after the `major` and `minor` version.

For some reason I cannot get the versioning to work for it. I've created a basic repository (https://github.com/gregoryduckworth/renovate-nginx) which shows the current state I am in. It has created a PR (https://github.com/gregoryduckworth/renovate-nginx/pull/1) but `release-` hasn't been stripped out with the `extractVersion`.

Am I missing something obvious or have I done something stupid?

**Comment by rarkins** [comment, marked as answer]
One problem is that you need to remove updateTypes from your package rule:

```
  "packageRules": [
    {
      "packageNames": ["nginx"],
      "extractVersion": "^release-(?<version>\\d+.\\d+)"
    }
  ],
```

=

**Reply by gregoryduckworth** [reply]
Thanks @rarkins, that has got me a lot closer! 

Now the only issue I have now is the return value, which comes back with the debug:
```
"updates": [
                   {
                     "fromVersion": "1.16",
                     "toVersion": "1.19",
                     "newValue": "1.19",
                     "newMajor": 1,
                     "newMinor": 0,
                     "updateType": "patch",
                     "isSingleVersion": true
                   }
                 ],
```

**Reply by rarkins** [reply]
Switch from your regex versioning to plain "docker" and it should work. But make sure you're running the very latest Renovate version as I fixed a bug you'll need after that.

**Reply by gregoryduckworth** [reply]
> Switch from your regex versioning to plain "docker" and it should work. But make sure you're running the very latest Renovate version as I fixed a bug you'll need after that.

Perfect @rarkins that solved it!
